src/app/lib/logger.node.js

```diff
@@ -35,6 +35,10 @@ const consoleLogFormat = printf(data => {
   return `${data.timestamp} ${data.level} ${JSON.stringify(logMessage)}`;
 });
 
+const expressFormatOptions = [prettyPrint(), colorize(), consoleLogFormat];
+
+const nextJSFormatOptions = [prettyPrint(), consoleLogFormat];
```

Comment:
Do these options have to be in this order? I just wondered if there might be a way to have a set of default formatting options, and then just adding on the ones which are extra?

```suggestion
const defaultFormatOptions = [prettyPrint(), consoleLogFormat];
const expressFormatOptions = [...defaultFormatOptions, colorize()];
```